mypy/build.py

```diff
@@ -860,8 +860,10 @@ def write_cache(id: str, path: str, tree: MypyFile,
 
     # Make sure directory for cache files exists
     parent = os.path.dirname(data_json)
-    if not os.path.isdir(parent):
+    try:
         os.makedirs(parent)
+    except os.error:
+        pass  # Assume it already exists.
```

Comment:
You can just call `os.makedirs(parent, exist_ok=True)` instead (as of 3.2, https://docs.python.org/3/library/os.html#os.makedirs).